src/notification.js

```diff
@@ -37,6 +44,29 @@ export class NotificationElement extends LitElement {
     this.readingLatestVersion = false;
     this.readingStableVersion = false;
     this.stableVersionAvailable = false;
+    // This will store information like user dimissing the notification. Any Notification sharing
+    // the same localStorageKey will be affected. If a specific Notification should not be dismissed after
+    // another has been dismissed, it requires a different localStorageKey
+    this.localStorageKey =
+      this.notificationStorageKey || "default-notification";
```

Comment:
I think it's better to define this as null, since we won't use this variable before calling `loadConfig` which will define the proper `localStorageKey`.

```suggestion
    this.localStorageKey = null;
```

Reply:
The reasoning behind this was to allow for custom `localStorageKey`. This way, a notification can be defined as "separate" from the default behaviour. For example if we had a notification that shouldn't be removed when the "PR build notification" was dismissed, we could define it as

```
<readthedocs-notification class="inverted" notification-storage-key="notification-about-something-else"></readthedocs-notification>
```

If that's not a feature we require, then I agree this is redundant.

Reply:
Yeah, we don't need this for now. We can add it in the future if we require it.